macros/anaUltraScurve.py

```diff
@@ -89,6 +89,8 @@
     myT.Branch( 'scurve_h', scurve_h)
     chi2 = array( 'f', [ 0 ] )
     myT.Branch( 'chi2', chi2, 'chi2/F')
+    Nev = array( 'f', [ 0 ] )
+    myT.Branch( 'Nev', Nev, 'Nev/F')
     pass
 
 vSum = {}
```

Comment:
Can also use the ```nesteddict``` here (basically for all ```dict```s, though the benefit comes only when they're actually nested)